lib/service/deeplink_service.dart

```diff
@@ -437,6 +437,11 @@ class DeeplinkServiceImpl extends DeeplinkService {
           _handleActivationDeeplink(activationID, _getOtpFromBranchData(data));
         }
         break;
+      case 'Autonomy_Connect':
```

Comment:
can we use `autonomy_connect`. if using snake case, don't need to capitalize the first letter of words.